x-pack/plugins/security_solution/common/endpoint/index_data.ts

```diff
@@ -401,15 +396,15 @@ const fleetEnrollAgentForHost = async (
     });
 
     if (res) {
-      const enrollObj: PostAgentEnrollResponse = await res.json();
+      const enrollObj = await res.json();
       if (!res.ok) {
         // eslint-disable-next-line no-console
         console.error('unable to enroll agent', enrollObj);
         return;
       }
       // ------------------------------------------------
       // now check the agent in so that it can complete enrollment
-      const checkinBody: PostAgentCheckinRequest['body'] = {
+      const checkinBody = {
```

Comment:
Thanks for these changes. We also have an issue open to fix this since enrolling a "fake" agent via kibana no longer works in our data generator.